checkov/terraform/context_parsers/tf_plan/tf_plan_json.py

```diff
@@ -317,14 +213,41 @@ def scan_once(string, idx):
 
     return _scan_once
 
+def find_indexes(s, ch='\n'):
+    """Finds all instance of given char and returns list of indexes """
+    return [i for i, ltr in enumerate(s) if ltr == ch]
+
+def count_occurrences(arr, key): 
+    """Binary search indexes to replace str.count """
+    n = len(arr)
+    left = 0
+    right = n - 1
+    count = 0
+ 
+    while (left <= right):
+        mid = int((right + left) / 2)
+
+        if (arr[mid] <= key):
+            count = mid + 1
+            left = mid + 1
+        else:
+            right = mid - 1
+    return count
+
+def largest_less_than(indexes, line_num, pos):
+    """Replacement func for python str.rfind using indexes """
+    return indexes[line_num-1] if count_occurrences(indexes, pos) and len(indexes) > 1 else -1
```

Comment:
```suggestion
    return indexes[line_num-1] if count_occurrences(indexes, pos) and indexes else -1
```
This should be a bit faster, since you don't need to calculate the length. I'm not sure, if it would be better to switch the `and` conditional sides, depends how often `indexes` is empty 🤔